dotmanager/constants.py

```diff
@@ -43,8 +43,14 @@
 import os
 import sys
 from dotmanager.types import Path
-from dotmanager.utils import normpath
+from dotmanager.utils import normpath, find_files
```

Comment:
No comma seperation in import statements. Also they should be ordered alphabetically